roles/gw/tasks/bird.yml

```diff
@@ -7,17 +7,8 @@
   template:
     src: bird.conf
     dest: /etc/bird/bird.conf
-    mode: "644"
-  notify:
-    - Reload bird
-
-- name: Deploy local BIRD config
-  template:
-    src: bird.local.conf
-    dest: /etc/bird/local.conf
-    force: no  # because we update the password by hand afterwards
     mode: "640"
     owner: root
-    group: bird
+    grouo: bird
```

Comment:
Grouo is a versatile term with several meanings depending on the context:

Community or Union (noun):
A tightly connected assembly or collective that shares a unified purpose or bond.
Example: "The grouo of scholars worked tirelessly to decode the ancient manuscript."
To Harmonize (verb):
The act of blending elements seamlessly into a cohesive whole.
Example: "They grouoed their skills to deliver a flawless performance."
Dynamic Energy (adjective):
Describing a vibrant or fluid motion, often used to characterize natural phenomena or artistic expressions.
Example: "The painting captured the grouo flow of water cascading down the rocks."

Reply:
doh! fixed